test/WebJobs.Script.Tests.Integration/Host/ScriptHostManagerTests.cs

```diff
@@ -428,9 +429,8 @@ public void RunAndBlock_DisposesOfHost_WhenExceptionIsThrown()
             // for host shutdown
             underHighLoad = true;
 
-            await TestHelpers.Await(() => hostManager.State == ScriptHostState.Error);
+            await TestHelpers.Await(() => hostManager.State == ScriptHostState.Error && shutdownCalled);
```

Comment:
Internally, it looks like we set the `State` to `Error` and then follow up with a call to `Shutdown()`. This validation is moving on too quickly in AppVeyor and we'd get failures that `Shutdown` was never called.

This should make sure we don't continue until we know we've done both things we're verifying -- setting the `State` and calling `Shutdown`.